source/macros.tex

```diff
@@ -156,6 +156,7 @@
 
 % Code and definitions embedded in text.
 \newcommand{\tcode}[1]{\CodeStylex{#1}}
+\newcommand{\mcode}[1]{\text{\tcode{#1}}}   % \tcode in math mode
```

Comment:
@godbyk: I don't think that's correct. There is no mathematical concept of subtracting iterators. However, we do have magic library wording that gives the minus-expression meaning with the library (even if it's not defined for the actual iterator type!).